test_data/refresh-testdata.sh

```diff
@@ -57,6 +57,62 @@ addq ${malcontent} --format=markdown \
 	macOS/2023.3CX/libffmpeg.dylib \
 	macOS/2023.3CX/libffmpeg.dirty.dylib
 
+addq ${malcontent} --format=markdown \
+	-o "${test_data}/macOS/2023.3CX/libffmpeg.change_increase.mdiff" \
+	--file-risk-change \
+	diff \
+	macOS/2023.3CX/libffmpeg.dylib \
+	macOS/2023.3CX/libffmpeg.dirty.dylib
+
+addq ${malcontent} --format=markdown \
+	-o "${test_data}/macOS/2023.3CX/libffmpeg.change_decrease.mdiff" \
+	--file-risk-change \
+	diff \
+	macOS/2023.3CX/libffmpeg.dirty.dylib \
+	macOS/2023.3CX/libffmpeg.dylib
+
+addq ${malcontent} --format=markdown \
```

Comment:
This technically does nothing, but the filtering works:
```
time=2024-10-10T20:56:45.979-05:00 level=INFO source=.../malcontent/pkg/action/diff.go:323 msg="dropping result because diff scores were the same" paths="out/samples-0ff28cbe99bc4610c58016faeb1a806a6e5cebbb/macOS/2023.3CX/libffmpeg.dylib (2) out/samples-0ff28cbe99bc4610c58016faeb1a806a6e5cebbb/macOS/2023.3CX/libffmpeg.dylib (2)"
time=2024-10-10T20:56:45.979-05:00 level=INFO source=.../malcontent/pkg/action/diff.go:323 msg="dropping result because diff scores were the same" paths="out/samples-0ff28cbe99bc4610c58016faeb1a806a6e5cebbb/macOS/2023.3CX/libffmpeg.dylib (2) out/samples-0ff28cbe99bc4610c58016faeb1a806a6e5cebbb/macOS/2023.3CX/libffmpeg.dylib (2)"
```

Reply:
It might make sense to do some diffs with different paths as well. I was testing that locally by copying the contents of `macOS` to a separate directory and then generating the diffs based on those paths.